# #-------------Accelerometer-----------
accel_misalignment: [1,-0.000893405,-0.00657313,
0,1,-0.00257073,
0,0,1]

accel_scale: [0.993735,0,0,
0,1.00372,0,
0,0,0.99558]

accel_bias: [-0.0584223,-0.223738,0.081999]

# #-------------Gyroscope-----------
gyro_misalignment: [1,-0.0143085,0.011295,
-0.0113609,1,-0.000702008,
-0.0589146,0.0211521,1]

gyro_scale: [1.0018,0,0,
0,0.996883,0,
0,0,1.01819]

gyro_bias: [-0.000627026,-0.00863939,0.002226]

